library/alloc/src/rc.rs

```diff
@@ -255,19 +255,27 @@ use core::convert::{From, TryFrom};
 use core::fmt;
 use core::hash::{Hash, Hasher};
 use core::intrinsics::abort;
+#[cfg(feature = "global-oom-handling")]
```

Comment:
I assume this is being added (as well as some other import cfgs) due to being unused after cfgs later in this file? Could we instead add allow(unused) or do, gated at crate root, enabled when the feature is off?